data/mods/Xedra_Evolved/spells/hedge_magic_eocs.json

```diff
@@ -122,35 +122,450 @@
               {
                 "u_message": "As you finish the words of the charm, the magic takes hold.  (E)xamine a nearby crop to fertilize it.",
                 "type": "good"
-              },
-              { "run_eocs": "EOC_HEDGE_MAKE_PLANTS_GROW_REMOVE", "time_in_future": 3 }
-            ],
-            "false_effect": [ { "u_message": "You must complete the charm within one hour of noon for it to be effective.", "type": "bad" } ]
-          }
+            }
```

Comment:
},
              { "run_eocs": "EOC_HEDGE_MAKE_PLANTS_GROW_REMOVE", "time_in_future": 3 }
            ],
            "false_effect": [ { "u_message": "You must complete the charm within one hour of noon for it to be effective.", "type": "bad" } ]
          }